### ISSUE TITLE
Improve output when alias is not found

### ISSUE THREAD

**Issue by shvaikalesh** [brunch/init-skeleton#23]

**Comment by denysdovhan** [comment]
What do you mean?

**Comment by ngottlieb** [comment]
I'm kind of wondering about this. I just submitted a skeleton and it got merged into the list, but I included an alias -- `leaflet` -- which won't work. This lead me to dig a little more and realize that the aliases will only work when a new release of `brunch-skeletons` is issued; it looks like that hasn't happened in about a year. And, additionally, people who have older versions of `brunch` won't be able to use the aliases either.

I'm going to look and see if I can think of a way around this, but I would agree with the original poster here. The current error is hard to interpret:

```
$ brunch new -s leaflet
13:14:41 - warn: Couldn't interpret "leaflet" as a hosted git url
13:14:41 - log: Cloning git repo "leaflet" to "/Users/ngottlieb/.brunch/skeletons/5f5555f44fb422ffc36bf68ec31937937f8d1ede"...
(node:1123) UnhandledPromiseRejectionWarning: Error: Git clone error: Error: Command failed: git clone leaflet "/Users/ngottlieb/.brunch/skeletons/5f5555f44fb422ffc36bf68ec31937937f8d1ede"
fatal: repository 'leaflet' does not exist
```

and instead maybe we could rescue this and suggest users use the full name of the skeleton instead.
